scripts/aggregate-themes/README.md

```diff
@@ -37,7 +37,7 @@ project
         ├── component-2.scss
         └── terra-dark-theme.scss
 ```
-*Note*: To support previous functionality, if a project contains existing `root` or `scoped` theme files within namespaced theme directories, terra-toolkit will aggregate these and skip generation. Toolkit will default to theme generation on the 6.0 release.
+*Note*: To support previous functionality, if a project contains existing `root.scss` or `scoped.scss` theme files within namespaced theme directories, terra-toolkit will aggregate those theme files first, then generate theme files second. Based on this import order, generated theme files take precedence over aggregated theme files. Toolkit will default to theme generation on the 6.0 release.
```

Comment:
> those theme files first, then generate theme files second.
This is a little confusing to me.  Why aggregate a project's theme first, but then the genearted files take precedence over the defined theme?

Reply:
I re-worded this, does that help clear the confusion? https://github.com/cerner/terra-toolkit/pull/376/commits/4c80666bf163c0a3509a771e3ce01ea969f1bb33#diff-a385b55d94aefec6207446ff49b50d96R40

What I forgot to mention is the CSS import order follows that order.
```js
import 'orion-fusion-theme/src/scss/root-theme.scss'; // existing root theme
import './root-orion-fusion-theme.scss'; // generated file. Takes precedence because it is imported after existing root theme.
```
On second thought, maybe I should include this example in the doc